### ISSUE TITLE
Configure toggle for cmd+e: workbench.action.openNextRecentlyUsedEditorInGroup 

### ISSUE THREAD

**Issue by chrisleeuber** [microsoft/vscode#82859]
Current behavior: 
On Mac: 
Pressing `Cmd+E` opens up `openNextRecentlyUsedEditorInGroup` action
Letting go of `Cmd` will result in the selected file being opened

Desired behavior:
Pressing `Cmd+E` opens up openNextRecentlyUsedEditorInGroup action
Letting go of `Cmd` leaves the file selection widget open
Pressing `return` will result in the selected file being opened

Reasoning for feature request: 
Other programs with `Cmd+E` functionality work the way I described (e.g. IntelliJ IDEA, Quick Tabs browser extension). 
It would be a better user experience if VS Code followed the same convention.
OR current behavior can be default but make this a configurable setting.

**Comment by bpasero** [comment]
Keybindings are configurable, I do not think I would change the default.

**Comment by chrisleeuber** [comment]
> Keybindings are configurable, I do not think I would change the default.

The issue is about making the behavior of the `Cmd+E` popup configurable. I am not requesting the default behavior to change, it should work as it currently does.

> The described behavior is how it is expected to work. If you disagree, please explain what is expected and what is not in more detail. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.
> 
> Happy Coding!

I was trying to be as descriptive as possible on what is the desired behavior. Let me try again.

Currently: When you press `Cmd+E` it brings up the `openNextRecentlyUsedEditorInGroup` popup. When you let go of `Cmd+E`, the selected file is immediately opened (you change the selection by pressing `Cmd+E` or `Shift+Cmd+E`.

Desired: An additional configuration, say `openNextRecentlyUsedEditorInGroupPopupBehavior: toggle`, which if does not exist will continue to work as it currently does. If this config does exist, then the behavior will be as follows
When you press `Cmd+E` it brings up the `openNextRecentlyUsedEditorInGroup` popup. When you let go of `Cmd+E`, the popup remains open. You can change the selected file by pressing `Up` or `Down` keyboard buttons. When you press `Enter` the selected file is opened and the popup disappears.

This is how IntelliJ currently works by default, but I understand the default behavior in VSCode has stuck, and a lot of people are used to how this works so changing the default now is not what I would want. However, for people who use this functionality a lot, and do use IntelliJ, VSCode and QuickTabs extension, having consistent behavior of `Cmd+E` when switching between the applications would lighten the cognitive burden just that little bit. Allowing this behavior to be configurable will allow such users a consistent experience.
